solutions/arrays/lucky-number

```diff
@@ -0,0 +1,59 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+function findLuckyNumber(nums){
+  // implement this function
+  
+
+// tle solution
+
+
+//   for(let index=0;index<nums.length;index++)
+//   {
+// 	  let count=1
+// 	  for(let index1=index+1;index1<nums.length;index1++)
+// 	  {
+// 		  if(nums[index]===nums[index1])
+// 		  {
+// 			  count=count+1
+// 		  }
+// 	  }
+// 	  if(count===nums[index])
+// 	  {
+// 		  return count
+// 	  }
+//   }
+//   return -1
+
```

Comment:
@DevratAgarwal  since the array is already sorted, we can do it with linear time complexity,

Reply:
@LVSsandeepVarma done

Reply:
@DevratAgarwal  I have updated the template code, please update it and we can do this problem in O(n) time  complexity

Reply:
@LVSsandeepVarma code is in O(n) time complexity